pywbemtools/pywbemcli/_cmd_server.py

```diff
@@ -412,3 +437,138 @@ def cmd_server_remove_mof(context, options):
         raise click.ClickException("Compile failed.")
     except Error as exc:
         raise pywbem_error_exception(exc)
+
+
+def cmd_server_schema(context, options):
+    """
+    The schema command provides information on the dmtf model schems and
+    schema versions installed in the server.
+    """
+    def experimental_display(value):
+        """Return string Experimental or empty sting"""
+        return 'Experimental' if value else ''
+
+    def version_str(version_tuple):
+        """Convert 3 integer tuple to string  (1.2.3) or empty strig"""
+        if all(i == version_tuple[0] for i in version_tuple):
+            return ""
+        return ".".join([str(i) for i in version_tuple])
+
+    conn = context.pywbem_server.conn
+    wbem_server = context.pywbem_server.wbem_server
+
+    output_format = validate_output_format(context.output_format, 'TABLE')
+    namespace_opt = options['namespace']
+
+    # Get namespaces. This bypasses the issue whene there is no interop
+    # namespace
+    try:
+        namespaces = [namespace_opt] if namespace_opt else \
+            wbem_server.namespaces
+    except ModelError:
+        namespaces = [wbem_server.conn.default_namespace]
+
+    rows = []
+    detail = options['detail']
+
+    for ns in sorted(namespaces):
+        klasses = conn.EnumerateClasses(namespace=ns, DeepInheritance=True,
+                                        LocalOnly=True)
+        classes_count = len(klasses)
+        ns_experimental = False
+        ns_max_version = [0, 0, 0]
+        # Schema names should be case independent
+        schemas = NocaseDict()
+        schema_max_ver = NocaseDict()
+        schema_experimental = NocaseDict()
+        no_schema = []
+
+        for klass in klasses:
+            this_class_experimental = False
+            # Determine if experimental qualifier exists
+            if ns_experimental is False:
+                if is_experimental_class(klass):
+                    ns_experimental = True
+                    this_class_experimental = True
+
+            schema_elements = klass.classname.split('_', 1)
+            schema = schema_elements[0] if len(schema_elements) > 1 \
+                else "{no schema}"
```

Comment:
In the fixed string that denotes all schemas, you have used parenthesis. Here, curly braces are used. I suggest to stick with one pair of characters only, e.g. parenthesis.

Reply:
I did that specifically so that the no schema entry would be sorted to the bottom of the list.  The alternate was more messy code and it looks better if that is at the bottom of the list.  using parenthesis puts that string at the top of the list.

FIXED